circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  node:
+    version: 6.5.0
+
+dependencies:
+  cache_directories:
+    - "node_modules"
+test:
+  override:
+    - npm run test:ci
+    - ./node_modules/.bin/codeclimate-test-reporter < ./coverage/lcov.info
```

Comment:
We used to use TravisCI for CI/CD .. if circle ci is free for open source stuff I don't mind staying with circle ci